lib/aemo/nmi.rb

```diff
@@ -603,13 +603,20 @@ def data_streams_by_status(status = 'A')
       @data_streams.select { |x| x.status == status.to_s }
     end
 
-    # The current daily load
+    # The current daily load in kWh
     #
     # @return [Integer] the current daily load for the meter
     def current_daily_load
       data_streams_by_status.map { |x| x.averaged_daily_load.to_i }.inject(0, :+)
     end
 
+    # The current annual load in MWh
+    #
+    # @return [Integer] the current annual load for the meter
+    def current_annual_load
+      (current_daily_load * 365.25 / 1000).to_i
```

Comment:
Probably, but AEMO has never experienced a year divisible by four that
wasn't a leap year so it's hard to tell.

On Tuesday, 14 June 2016, Joel Courtney <notifications@github.com> wrote:

> In lib/aemo/nmi.rb
> <https://github.com/jufemaiz/aemo/pull/15#discussion_r66901492>:
>
> >      #
> >      # @return [Integer] the current daily load for the meter
> >      def current_daily_load
> >        data_streams_by_status.map { |x| x.averaged_daily_load.to_i }.inject(0, :+)
> >      end
> >
> > +    # The current annual load in MWh
> > +    #
> > +    # @return [Integer] the current annual load for the meter
> > +    def current_annual_load
> > +      (current_daily_load * 365.25 / 1000).to_i
>
> Should this be 365.2425 not 365.25?
>
> https://en.wikipedia.org/wiki/Year
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jufemaiz/aemo/pull/15/files/6870ba24f31947d576518e6395b47f225a51140b#r66901492>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AH2eN7i_1qrOyD6f1tDCBgEFIzRV7nBFks5qLhPtgaJpZM4IfEpn>
> .
>